javascript/ql/src/Security/CWE-022/ZipSlipBad.js

```diff
@@ -0,0 +1,9 @@
+const fs = require('fs');
+const unzip = require('unzip');
+
+fs.createReadStream('archive.zip')
+  .pipe(unzip.Parse())
+  .on('entry', entry => {
+    const fileName = entry.path;
```

Comment:
This constant is not used anywhere, is it? (Same below.)

Reply:
Oof, excellent point, fixed this throughout documentation good and bad examples, and test good and bad examples, and changed the test expected results to match. I think they match my intent in that `...Bad` and `...Bad2` occur in the `#select`.